spec/lib/pre_assembly/batch_spec.rb

```diff
@@ -251,8 +251,8 @@
 
   describe '#discover_items_via_crawl' do
     it 'returns expected information' do
-      items = %w[abc.txt def.txt ghi.txt 123.tif 456.tif 456.TIF].map { |i| flat_dir_images.bundle_dir_with_path i }
-      allow(flat_dir_images).to receive(:dir_glob).and_return(items)
+      # these are the actual files in the spec/test_data/flat_dir_images bundle directory
+      items = %w[checksums.txt image1.tif image2.tif image3.tif manifest.csv manifest_badsourceid_column.csv].map { |i| flat_dir_images.bundle_dir_with_path i }
       expect(flat_dir_images.send(:discover_items_via_crawl, flat_dir_images.bundle_dir)).to eq(items.sort)
```

Comment:
this test was oddly mocking the expected output it was then testing... let's actually let `Dir.glob` run and discover the actual files (which were different than what we were mocking before)

Reply:
yeah, well, a lot of devs learned how to write test code on the job, and I think the origins of this app were the 4 revs hires (you, Monty, ...)

Reply:
didn't mean my comment as a critique of anyone in particular (as you note, it could have been my original code) -- just trying to make it clear why changing the expected output was valid in this case